content edit to steps page: Intro and Step one (#371)
DFE-Digital/get-into-teaching-content: content/steps-to-become-a-teacher.md

## Become a qualified teacher in England

Follow these steps if you want to become a qualified primary or secondary school teacher in England. You'll need to work towards Qualified Teacher Status (QTS). This can be achieved in different ways, including:

* doing a postgraduate initial teacher training (ITT) course if you already have a degree
* doing an undergraduate degree course which leads to QTS

> If you're interested in teaching a different age group, consider training to teach [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).

Reviewer: I'm not sure 'different age group' works here - it's not clear what they're different to. Could it perhaps be more explicit and say "younger children or 16+" (obviously worded more nicely!)

Reviewer 2: Agree - I'll push a change

Author: ```suggestion
If you're not looking to teach primary or secondary, consider training to teach [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
```

Author: Thanks @peteryates. I've added in a temporary fix.

* [becoming a further education teacher](/further-education-teacher-training)
* [becoming an early years teacher](/early-years-teaching-training)
If you're interested in teaching a different age group, consider training to teach [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).

To work as a qualified primary or secondary school teacher in a state school in England, you need to
have Qualified Teacher Status (QTS).

There are a number of ways to get QTS, such as:

* completing a postgraduate initial teacher training (ITT) course
* doing a degree course which leads to QTS

We also have information for:

> * [international candidates](/international-candidates)

Reviewer 2: @EmmaFrith what's the logic for losing International candidates links?

Author: I was thinking these steps are actually the same for international candidates. And that the international page needs some more work to make it more digestible.

Author: Was wondering whether, when we review the steps, we link off to more detailed international guidance on the relevant steps themselves, e.g. quals, rather than off-ramping a whole group of people (who can actually follow the same steps as everyone else)
